packages/charts/src/components/distributed-chart-tooltip/utils/get-distributed-chart-tooltip-settings.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  formatValue as valueFormatter,
+  getFromPath,
+  getKeysDifference,
+  ScaleSettings,
+  TooltipFormatter,
+  transformToPercent,
+} from '@keen.io/charts-utils';
+
+import { DataSelector } from '../../../types';
+import { getLabel } from '../index';
+
+type TooltipSettings = {
+  data: Record<string, any>[];
+  keys: string[];
+  disabledKeys: string[];
+  selectors: { selector: DataSelector; color: string }[];
+  isPercentage: boolean;
+  labelSelector: string;
+  formatValue?: TooltipFormatter;
+  scaleSettings?: ScaleSettings;
+};
+
+const getDistributedChartTooltipSettings = ({
+  data,
+  keys,
+  disabledKeys,
+  selectors,
+  isPercentage,
+  labelSelector,
+  formatValue,
+  scaleSettings,
```

Comment:
set default value for scaleSettings `{}` as it's optional and it's destructured later